SparkPost/2web2ui
# AC-734: Plan picker uses tiered dropdown

[Wireframe](https://jira.int.messagesystems.com/browse/UX-91?focusedCommentId=491696&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-491696)

### What Changed
 - Separates plans with tiers into "Test Account", "Starter", or "Premier"
 
### How To Test
 - Open `/account/billing/plan` or `/onboarding/plan` and check that the new dropdown uses those tiers
 - Check that billing flow still works correctly with selecting new plans

### To Do
- [x] Update styling to UX
- [x] Separate tier to constants file
- [x] Layer list on top of selected plan
- [x] Add testing for Downshift render function
